help/destinations/catalog/advertising/acxiom-prospect-suppression.md

```diff
@@ -0,0 +1,165 @@
+
+![Existing Account](../../assets/catalog/advertising/acxiom/image-destination-account.png)
+
+### Fill in destination details {#destination-details}
+
+To configure details for the destination, fill in the required and optional fields below. An asterisk next to a field in the UI indicates that the field is required.
+
+![Destination Detail](../../assets/catalog/advertising/acxiom/image-destination-details.png)
+
+* **Name (Required)** - The name the destination will be saved under
+* **Description** -  Short explanation of the destination's purpose
+* **Bucket Name (Required)** - Name of the Amazon S3 bucket set up on S3
+* **Folder Path (Required)** - If subdirectories in a bucket are used a path must be defined, or '/' to reference the root path.
+* **File Type** - Select the format Experience Platform should use for the exported files. Currently, the only file type Acxiom processing will be expecting is CSV 
+
+>[!IMPORTANT]
+>
+>When selecting the CSV option, *Delimiter*, *Quote Character*, *Escape Character*, *Empty Value*, *Null Value*, *Compression format*, and *Include menifest file* options will be presented, the following document explains these settings in more detail [configure the formatting options](../../ui/batch-destinations-file-formatting-options.md).
+
+![CSV Options](../../assets/catalog/advertising/acxiom/image-destination-csv-options.png)
+
+### Enable alerts {#enable-alerts}
+
+You can enable alerts to receive notifications on the status of the dataflow to your destination. Select an alert from the list to subscribe to receive notifications on the status of your dataflow. For more information on alerts, see the guide on [subscribing to destinations alerts using the UI](../../ui/alerts.md).
+
+When you are finished providing details for your destination connection, select **[!UICONTROL Next]**.
+
+## Activate audiences to this destination {#activate}
+
+>[!IMPORTANT]
+>
+>* To activate data, you need the **[!UICONTROL View Destinations]**, **[!UICONTROL Activate Destinations]**, **[!UICONTROL View Profiles]**, and **[!UICONTROL View Segments]** [access control permissions](/help/access-control/home.md#permissions). Read the [access control overview](/help/access-control/ui/overview.md) or contact your product administrator to obtain the required permissions.
+>* To export *identities*, you need the **[!UICONTROL View Identity Graph]** [access control permission](/help/access-control/home.md#permissions). <br> ![Select identity namespace highlighted in the workflow to activate audiences to destinations.](/help/destinations/assets/overview/export-identities-to-destination.png "Select identity namespace highlighted in the workflow to activate audiences to destinations."){width="100" zoomable="yes"}
+
+Read [Activate audience data to batch profile export destinations](/help/destinations/ui/activate-batch-profile-destinations.md) for instructions on activating audiences to this destination.
+
+### Mapping suggestions
```

Comment:
thanks for adding this section! can we please clarify in written -- "Processing requires name and address elements."

^^ does this mean that name and address fields are mandatory in order to ensure that is properly processed on the Acxiom side? if yes, please make this very clear in the documentation.

Reply:
Reformatted this to try to clarify the requirements; "Processing requires name and address elements, while not all elements are required providing as much as possible will aid in successful matching."